src/sentry/static/sentry/app/views/performance/transactionDetails/quickTraceMeta.tsx

```diff
@@ -27,6 +27,7 @@ function handleTraceLink(organization: OrganizationSummary) {
     eventKey: 'quick_trace.trace_id.clicked',
     eventName: 'Quick Trace: Trace ID clicked',
     organization_id: parseInt(organization.id, 10),
+    source: 'events',
```

Comment:
Did you mean to have this say something like `issue` or `performance` instead of just a hard-coded value?

Reply:
Ahh I misread, issues have a separate handler.